lib/utils.js

```diff
@@ -644,23 +659,14 @@ function deprecateOptions(config, fn) {
   return deprecated;
 }
 
-const SUPPORTS = {};
-// Test asyncIterator support
-try {
-  require('./async/async_iterator');
-  SUPPORTS.ASYNC_ITERATOR = true;
-} catch (e) {
-  SUPPORTS.ASYNC_ITERATOR = false;
-}
-
 class MongoDBNamespace {
   constructor(db, collection) {
     this.db = db;
     this.collection = collection;
   }
 
   toString() {
-    return this.collection ? `${this.db}.${this.collection}` : this.db;
+    return this.collection ? `${this.db}.${this.collection}` : `${this.db}`;
```

Comment:
is this change required because `this.db` doesn't have type information? Would it still be required if jsdoc's were added for this type with correct type information?

Reply:
Yea JSDoc fixes it, good catch, that is certainly preferable